tests/contact-controller.test.js

```diff
@@ -439,11 +556,14 @@ test('Call findAll(queryObject) with singleInput and ignore other params', async
     firstName: 'Jill',
     lastName: 'Smith',
     counselor: 'counselorId',
-    phoneNumber: 'Anonymous',
+    // phoneNumber: '123', // currently this case is not handled
```

Comment:
Oh, I misunderstood when you asked me about this.  Even though we shouldn't ever get this query from our frontend, it's a legal query to our API, so we shouldn't just comment it out and hope for the best.  It looks like the expected behavior of the API as defined by the test is to ignore all other params if `singleInput` is present, so let's make that behavior work for `phoneNumber` just like the other params.